tests/integration/auth/keeper/account_retriever_test.go

```diff
@@ -1,16 +1,17 @@
-package keeper
+package keeper_test
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
+	authTest "github.com/cosmos/cosmos-sdk/tests/integration/auth/keeper"
```

Comment:
is there any reason for having `AppConfig` in a different package?? looks like some have it in the same package while others don't  🤷‍♂️

Reply:
It's in the same package. Tests are with `keeper_test` package name and `app_config` is with keeper package name.